Intersect.Client/Core/Controls/ControlEnum.cs

```diff
@@ -64,6 +64,8 @@ public enum Control
 
         OpenAdminPanel,
 
+        OpenGuild,
```

Comment:
Any particular reason this isn't at the end of the enum?

Reply:
No reason at all

Reply:
Should we move it above OpenSettings so it is with the other opening of game windows (Inventory/Spells/Quests/etc)?

Reply:
Probably yeah. I don't think the config uses the enum numbering but uses the names instead so it shouldn't make a difference where it's located.